crates/ruff_diagnostics/src/fix.rs

```diff
@@ -123,4 +123,11 @@ impl Fix {
     pub fn applicability(&self) -> Applicability {
         self.applicability
     }
+
+    /// Sort the [`Edit`] elements in the [`Fix`] by their [`Edit::start`] position.
+    #[must_use]
+    pub fn sorted(mut self) -> Self {
+        self.edits.sort_by_key(Edit::start);
+        self
+    }
```

Comment:
Open to suggestions, but given that we're generating three edits for the fix, and they can come in almost any order, this feels like the easiest solution.

Reply:
https://github.com/charliermarsh/ruff/pull/4762